### ISSUE TITLE
pass back 'GPString' output

### ISSUE THREAD

**Issue by jgravois** [jgravois/esri-leaflet-gp#28]
something slightly more elaborate than this:
```js
  // if the output is a file, pass it along 'as is'
  if (response.dataType === 'GPDataFile') {
    processedResponse.outputFile = response.results[0];
  }
  // otherwise return the GPString output 'as is'
  else {
    processedResponse = response;
  }
```

**Comment by jgravois** [comment]
for my mock GP Server

```python
import arcpy

outString = arcpy.setParameterAsText(0,"all done")
outBoolean = arcpy.setParameterAsText(0, True)
```
with corresponding 'derived/output` params in the GP dialog...

**Comment by jgravois** [comment]
closed via 4e6bed027278d58fe49fc2870209f9123790f155
